tests/test_VertexPartition.py

```diff
@@ -175,8 +175,10 @@ def test_Bipartite(self):
     graph = bipartite_graph
     partition, partition_0, partition_1 = \
         leidenalg.CPMVertexPartition.Bipartite(graph, resolution_parameter_01=0.2)
-    self.optimiser.optimise_partition_multiplex([partition, partition_0, partition_1],
-                                     layer_weights=[1, -1, -1])
+    self.optimiser.optimise_partition_multiplex(
+            [partition, partition_0, partition_1],
+            layer_weights=[1, -1, -1],
+            fixed_nodes=None)
```

Comment:
It shouldn't be necessary to add the `fixed_nodes` argument, right?

Reply:
This is an inconsistency in the preexisting interface, since optimise_partition_multiplex does not deal with keyword arguments. I'll fix that to be consistent with the rest of the code, but did not want to start fixing bugs around without a quick discussion,